Build-V3-Plugin-Monitoring-ios.podspec

```diff
@@ -7,7 +7,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'Build-V3-Plugin-Monitoring-ios'
-  s.version          = '1.0.2-beta.1'
+  s.version          = '1.0.2'
```

Comment:
This need to become 1.0.3-beta.1. We have to do it manually since V3 plugins are not automated like V4